plugins/org.eclipse.fordiac.ide.application/src/org/eclipse/fordiac/ide/application/figures/FBNetworkConnection.java

```diff
@@ -298,8 +298,8 @@ int getMaxFanOutLabelWidth() {
 		final InterfaceEditPart source = (InterfaceEditPart) connEP.getSource();
 		return ((List<ConnectionEditPart>) source.getSourceConnections()).stream()
 				.filter(conn -> !conn.getModel().isVisible())
-				.map(ep -> ep.getFigure().getSourceDecoration().getLabel().getBounds().width).max(Integer::compare)
-				.orElse(0);
+				.map(ep -> Integer.valueOf(ep.getFigure().getSourceDecoration().getLabel().getBounds().width))
+				.max(Integer::compare).orElse(Integer.valueOf(0)).intValue();
```

Comment:
Please use an `IntStream` here:
```suggestion
				.mapToInt(ep -> ep.getFigure().getSourceDecoration().getLabel().getBounds().width).max()
				.orElse(0);
```